app/models/orm/migrations/draft_versions/04fcb4f2408a_add_metadata_table.py

```diff
@@ -0,0 +1,291 @@
+"""add metadata table.
```

Comment:
do we need this draft version one?

Reply:
I'll remove this. Created this temporarily to check in alembic scripts without doing the db change.